core/modules/system/css/system.theme.css

```diff
@@ -8,13 +8,15 @@
  * HTML elements.
  */
 fieldset {
-  min-width: 0; /* Webkit/Blink fix for mobile */
+  min-width: 0; /* Webkit/Blink fix for mobile, see stackoverflow.com/questions/17408815 */
```

Comment:
...with the addition of the SO link, the line goes beyond the 80 chars limit. Comment should go in a separate line on its own, above the ```min-width: 0;```